pkgs/applications/window-managers/eww/default.nix

```diff
@@ -6,6 +6,8 @@
 , gdk-pixbuf
 , withWayland ? false
 , gtk-layer-shell
+, librsvg
+, wrapGAppsHook
```

Comment:
please move `wrapGAppsHook` to after `pkg-config`, and `librsvg` to before `withWayland`

> Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first. (Last line in `CONTRIBUTING.md`)